extension/httpforwarder/extension_test.go

```diff
@@ -52,6 +53,7 @@ func TestExtension(t *testing.T) {
 		requestErrorAtForwarder     bool
 		clientRequestArgs           clientRequestArgs
 		startUpError                bool
```

Comment:
createError?

Reply:
I didn't change the existing test field names (it was originally 'startUpError'). I'm happy to refactor the existing fields if you would like and fix the structure for how the startup error is checked.?

Reply:
Ii think `statupError` is fine since it is actually the error returned by `Start`. @bogdandrutu did you mean something else?